lib/collection/src/collection.rs

```diff
@@ -76,6 +77,7 @@ pub struct Collection {
     pub(crate) id: CollectionId,
     pub(crate) shards_holder: Arc<LockedShardHolder>,
     pub(crate) config: Arc<RwLock<CollectionConfig>>,
+    pub(crate) collection_config: Arc<SharedStorageConfig>,
```

Comment:
This is confusing as hell. `config` field of type `CollectionConfig` and `collection_config` field of type `SharedStorageConfig`. At least, let's rename `collection_config` field to `storage_config`.

Reply:
missed this one, thanks